Sources/Lotsawa/Array+Extensions.swift

```diff
@@ -1,6 +1,7 @@
 extension Array {
-  /// Version of reserveCapacity that ensures repeated increasing requests have
-  /// amortized complexity O(N), where N is the total capacity reserved.
+  /// Version of `reserveCapacity` that ensures repeated increasing requests
+  /// have amortized complexity O(`N`), where `N` is the total capacity
```

Comment:
```suggestion
  /// have amortized complexity O(N), where N is the total capacity
```

I don't think it's necessary to put N in code voice here; it's just a mathematical quantity, not a program entity.

Reply:
I see. I copied the convention for ``O(`count`)`` from below:

https://github.com/val-lang/Lotsawa/blob/4720e9a4066c3de17f75c820cbee01d9a8592ca1/Sources/Lotsawa/Collection%2BExtensions.swift#L10

Your point that N (mathematical quantity) and `count` (program entity) being different makes sense.